### ISSUE TITLE
LazyLoaded modules packs modules multiple times ?

### ISSUE THREAD

**Issue by RoyiNamir** [NativeScript/nativescript-dev-webpack#395]
I've created a simple `tns create aaa --ng` project. ( ng5  , 3.4.1).
Then I've added a simple module to be loaded lazily : 

Added this to routing in main module: 
```
   {
        path:         "m",
        loadChildren: "./m1/m1.module#M1Module"
    }
```
Looking at `tns build android --bundle --env.uglify --env.report`

I get this : 

![image](https://user-images.githubusercontent.com/2842123/34610883-befcf95c-f22b-11e7-9330-0fb691a8a2e9.png)

I already know that if I `--env.aot` then the `compiler.js` will go away.

But still  , even if I remove the `compiler.js` by `--aot` , why does it is not in the common chunks ?

Which leads me to ask if it's a bug or by design ? 
I also see `renderer.js` in both modules.

Is there a solution for this ? or is it a bug ?

**Comment by jogboms** [comment]
@RoyiNamir any headway as to why you closed this?

**Comment by RoyiNamir** [comment]
@jogboms Because to tell you the truth , I give up. I don't know if it's a webpack problem / or webpack-dev native script problem or nativescript problem and i'm really frustrated about that.

And when I've asked webpack repo to see this : 

https://github.com/webpack/webpack/issues/6251

I've got an answer : see the docs. ( which I did see)
:-( 

Anyway I've asked in SO also.
https://stackoverflow.com/questions/48117888/webpacks-commonchunksplugin-configuration-with-lazy-loaded-modules
